packages/grub2/build.yaml

```diff
@@ -16,13 +16,30 @@ steps:
   {{end}}
 {{end}}
 
+{{if eq .Values.name "grub2-efi-image"}}
+- mkdir -p /EFI/BOOT
+- cp config/grub_efi.cfg /EFI/BOOT/grub.cfg
+  {{if .Values.distribution}}
+    {{if eq .Values.distribution "opensuse"}}
+- cp /usr/share/grub2/x86_64-efi/grub.efi /EFI/BOOT/bootx64.efi
+
+    {{else if eq .Values.distribution "fedora"}}
+- grub2-mkimage -O x86_64-efi -o /EFI/BOOT/bootx64.efi -c /EFI/BOOT/grub.cfg -p /grub2 -d /usr/lib/grub/x86_64-efi {{.Values.efi_modules}} 
+
+    {{else if eq .Values.distribution "ubuntu"}}
+- grub-mkimage -O x86_64-efi -o /EFI/BOOT/bootx64.efi -c /EFI/BOOT/grub.cfg -p /grub2 -d /usr/lib/grub/x86_64-efi {{.Values.efi_modules}} linuxefi
+
+    {{end}}
+  {{end}}
+{{end}}
+
 {{if eq .Values.name "grub2-artifacts"}}
 - mkdir /grub-artifacts
   {{if .Values.distribution}}
     {{if eq .Values.distribution "opensuse"}}
 - cp -rf /usr/share/grub2/* /grub-artifacts
     {{else}}
-- cp -rf /usr/share/grub/* /grub-artifacts
+- cp -rfL /usr/lib/grub/* /grub-artifacts
```

Comment:
@mudler I had to include `-L` because in Ubuntu there is symlink here under this path and the when installing/extracting the resulting package at install time luet was complaining of a broken symlink. Is this the intended behavior? If so I'd say it should fail at build time rather than install time.

Reply:
If the package ships a symlink, then luet will recreate a symlink when running `luet install`, but if it points to a path that doesn't exist then it shows a warning, but it will nevertheless recreate it.

So it really depends on the wanted behavior, luet will just try to reconstruct what it finds out

Reply:
well it did print stuff in stdout or stderr, but return code was not 0, so the builder script was failing, I did not actually verify if the installation happened or not. I guess it did.